flaskblog/__init__.py

```
@@ -18,7 +18,22 @@
 
 
 
-def create_app(config_class = Config):
+def create_app(config_class=Config):
```

Ruff (0.13.3)

Unused function argument: `config_class`

(ARG001)